tests/test_wave_03.py

```diff
@@ -54,13 +50,20 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********from HP
+    # expected_movies = [INTRIGUE_1, INTRIGUE_2, INTRIGUE_3]
+    # assert set(friends_unique_movies) == set(expected_movies)
+    
+    # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********from AT
+    expected_movies_titles = ["Zero Dark Python", 
+                              "The Programmer: An Unexpected Stack Trace", 
+                              "It Came from the Stack Trace"]
+    friends_unique_movie_titles = []
+    for movie in friends_unique_movies:
+        friends_unique_movie_titles.append(movie["title"])
+    assert set(expected_movies_titles) == set(friends_unique_movie_titles) #corrections added for wave 03 assertions
```

Comment:
When it comes to adding assertions to a test, we want our asserts to follow the structure of the output our functions are expecting. The `get_friends_unique_watched()` function should return a list of movies, not a set. With that in mind, we should asserting that the list we get is the list we expect rather than convert everything into sets.

When we write tests, we know exactly what the output should look like given the input, so we can use that to our advantage. Quickly looking through the test constants, we can see that the movies [FANTASY_4, HORROR_1, INTRIGUE_3] are the unique movies the user's friends have watched. In this case, it's a good idea to simply check that our output is the list we are expecting. 

```suggestion
    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```

We could alternately check to see if each particular movie is in the `friends_unique_movies` list.